.github/workflows/branch.yml

```diff
@@ -21,7 +22,15 @@ jobs:
         DISCORD_GUILD_ID: ${{ secrets.DISCORD_GUILD_ID }}
         DISCORD_SESSION_SECRET: ${{ secrets.DISCORD_SESSION_SECRET }}
         DISCORD_REDIRECT_URI: ${{ secrets.DISCORD_REDIRECT_URI }}
+    - run: (cd deploy && ./client_config_ci.sh)
+      env:
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        ENVIRONMENT: ${{ secrets.ENVIRONMENT }}
     - run: node common/scripts/install-run-rush.js lint
     - run: node common/scripts/install-run-rush.js lint:md
     - run: node common/scripts/install-run-rush.js test
     - run: node common/scripts/install-run-rush.js test:cypress --verbose
+    - run: node common/scripts/install-run-rush.js deploy --create-archive deploy.zip
+    - run: (cd deploy && npm run upload ${{ secrets.ENVIRONMENT }})
+    - run: (cd deploy && npm start ${{ secrets.ENVIRONMENT }})
```

Comment:
- [ ] `branch` and `main` are identical except for `environment` stanza, dedupe?

Reply:
later